conn.go

```diff
@@ -1055,6 +1055,32 @@ func (c *Conn) Create(path string, data []byte, flags int32, acl []ACL) (string,
 	return res.Path, err
 }
 
+func (c *Conn) CreateContainer(path string, data []byte, flags int32, acl []ACL) (string, error) {
+	if err := validatePath(path, flags&FlagSequence == FlagSequence); err != nil {
+		return "", err
+	}
+	if flags&FlagTTL != FlagTTL {
+		return "", ErrInvalidFlags
+	}
+
+	res := &createResponse{}
+	_, err := c.request(opCreateContainer, &CreateContainerRequest{path, data, acl, flags}, res, nil)
+	return res.Path, err
+}
+
+func (c *Conn) CreateTTL(path string, data []byte, flags int32, acl []ACL, ttl time.Duration) (string, error) {
+	if err := validatePath(path, flags&FlagSequence == FlagSequence); err != nil {
+		return "", err
+	}
+	if flags&FlagTTL != FlagTTL {
+		return "", ErrInvalidFlags
+	}
+
+	res := &createResponse{}
+	_, err := c.request(opCreateTTL, &CreateTTLRequest{path, data, acl, flags, ttl.Nanoseconds() / 1e6}, res, nil)
```

Comment:
you should be able to here do
```
ttl * time.Millisecond
```

we should not need to do any math on this unit to get the underlying int64 that is backing the time.Duration.

Reply:
Here I think I can modify to “int64(ttl / time.Millisecond)”